core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -86,6 +86,10 @@
 #error "Unknown platform flavor"
 #endif
 
+#define CSU_DMA_BASE		0xFFC80000
+#define CSU_DMA_SIZE		0x1000
+#define CSU_BASE		0xFFCA0000
```

Comment:
Tried to look for CSU size but did not spot it easily. Might be in some vivado generated headers.

Largest public register offset is 0x0000005034 (+4).

It might be a good idea to have size for CSU also defined. Can you spot it somewhere?

Reply:
yeah I had the same problem, I dont know...so I pretty much picked the addresses that I needed because 5038 seems weird...